src/middleware/tests/ldp/ldp.test.js

```diff
@@ -123,6 +126,31 @@ describe('CRUD Project', () => {
     );
   }, 20000);
 
+  test('Replace One Project', async () => {
+    const urlParamsPut = {
+      resource: {
+        '@context': {
+          '@vocab': 'http://virtual-assembly.org/ontologies/pair#'
+        },
+        '@id': projet1['@id'],
+        description: 'myProjectUpdated'
+      },
+      accept: MIME_TYPES.JSON,
+      contentType: MIME_TYPES.JSON
+    };
+    const updatedProject = await broker.call('ldp.resource.put', urlParamsPut);
+    expect(updatedProject['pair:description']).toBe('myProjectUpdated');
+    expect(updatedProject['pair:label']).toBe(undefined);
+    console.log(updatedProject);
+
+    const updatedPersistProject = await broker.call('ldp.resource.get', {
+      accept: MIME_TYPES.JSON,
+      resourceUri: projet1['@id']
+    });
+    expect(updatedPersistProject['pair:description']).toBe('myProjectUpdated');
+    expect(updatedProject['pair:label']).toBe(undefined);
```

Comment:
Jest documentation says it's better practice to use `.toBeUndefined` https://jestjs.io/docs/en/expect#tobeundefined

Reply:
ok